src/routes/connect/connectStore.ts

```diff
@@ -0,0 +1,5 @@
+import { writable } from 'svelte/store';
+
+export const isPrivateKeyVisible = writable(false);
+export const setInputValue = writable('');
+export const setConnectionError = writable('');
```

Comment:
these have function names, yet they are not functions. Simply call them inputValue and connectionError